MCprep_addon/spawner/effects.py

```diff
@@ -31,7 +37,9 @@
 
 from . import spawn_util
 
-from ..conf import env
+from ..conf import env, VectorType
+
+ListEffectsAssets = TypeVar("ListEffectsAssets") # Check spawn_util.py ListEffectsAssets
```

Comment:
Put the comment before the line, otherwise it'll be too long in an editor. I would also expand this comment a little
```suggestion
# Check spawn_util.py for the
# definition of ListEffectsAssets
ListEffectsAssets = TypeVar("ListEffectsAssets")
```